pkg/cvo/internal/operatorstatus.go

```diff
@@ -128,10 +132,11 @@ func waitForOperatorStatusToBeDone(ctx context.Context, interval time.Duration,
 		actual, err := client.Get(ctx, expected.Name)
 		if err != nil {
 			lastErr = &payload.UpdateError{
-				Nested:  err,
-				Reason:  "ClusterOperatorNotAvailable",
-				Message: fmt.Sprintf("Cluster operator %s has not yet reported success", expected.Name),
-				Name:    expected.Name,
+				Nested:            err,
+				UpgradeSucceeding: true,
```

Comment:
This is "Kube API-server down/unreachable or requested ClusterOperator is missing".  That all seems serious enough to be `UpgradeSucceeding: false`, although I guess it might be worth a safety valve to avoid freaking out on temporary networking/Kube-API-server hiccups.  Do we want something more nuanced in error type checking here?

Reply:
Do we want to make this UpdateEffectFailAfterInterval?